### DISCUSSION TITLE
Pigparse 3.24.88.3 released (today?) getting massive infection flags by bitdefender and malicious activity termination

### DISCUSSION THREAD

**Discussion by princealexi25** [smasherprog/EqTool#146] [General]
I can no longer use this program on my system as-is.  I'm going to search for an old install file and turn off auto update, if I can...

**Comment by smasherprog** [comment]
virus scanners are upset because i dont sign my exe any more.
it will be a while before i can reimplement a process for this.

**Comment by smasherprog** [comment]
the program is now signed which should help with this issue. Try the latest and let me know if it fixes the issue

**Comment by smasherprog** [comment]
going to close this it should be resolved. Please, let me know if it wasnt fixed
